### PR TITLE
NINOA-129-Update headers + responses

### PR DESCRIPTION
**Pull request by DurgaJonnalagadda**

### CODE REVIEW

File: resources/public/api/conf/1.0/headers.raml [hunk 1]
```diff
@@ -18,3 +18,31 @@ traits:
         type: string
         required: true
         example: 'application/vnd.hmrc.1.0+json'
+  acceptJson:
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/headers.raml]: Can you remove this header as I believe we add this header

File: resources/public/api/conf/1.0/headers.raml [hunk 2]
```diff
@@ -18,3 +18,31 @@ traits:
...
+    headers:
+      Authorization:
+        description: Authorization header based on OAuth2 token
+        type: string
+        required: true
+        example: 'Bearer QWA8BdXvGvQjqf2c4Cmql81yuW4a'
+  acceptJson:
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/headers.raml]: Again can you remove this header as I believe we add this header

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 3]
```diff
@@ -6,34 +6,36 @@
   "birthDateVerification": "VERIFIED",
   "officeNumber": "1234",
   "contactNumber": "1234567890",
-  "country": 1,
-  "name": {
+  "applicantNames": {
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing to 'names'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 4]
```diff
@@ -6,34 +6,36 @@
...
+    "nameType":"REGISTERED",
     "title": "MR",
     "forename": "AForename",
     "secondForename": "NotSure",
     "surname": "ASurname",
     "startDate": "10-10-2020",
     "endDate": "10-10-2020"
   },
-  "historicNames": [
+  "applicantHistoricNames": [
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: this needs changing to 'historicNames'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 5]
```diff
@@ -6,34 +6,36 @@
...
     {
+      "nameType":"REGISTERED",
       "title": "MRS",
       "forename": "AForename",
-      "secondForename": "NotSure",
+      "middleName": "NotSure",
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'secondForename'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 6]
```diff
@@ -6,34 +6,36 @@
...
       "surname": "ASurname",
       "startDate": "10-10-2020",
       "endDate": "10-10-2020"
     },
     {
+      "nameType":"REGISTERED",
       "title": "MISS",
       "forename": "AForename",
-      "secondForename": "NotSure",
+      "middleName": "NotSure",
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'secondForename'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 7]
```diff
@@ -6,34 +6,36 @@
...
       "surname": "ASurname",
       "startDate": "10-10-2020",
       "endDate": "10-10-2020"
     }
   ],
-  "address": {
+  "applicantAddresses": {
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing to 'addresses'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 8]
```diff
@@ -45,7 +47,7 @@
     "startDate": "10-10-2020",
     "endDate": "10-10-2020"
   },
-  "historicAddresses": [
+  "applicantHistoricAddresses": [
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'historicAddresses'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 9]
```diff
@@ -71,11 +73,10 @@
       "endDate": "10-10-2020"
     }
   ],
-  "originData": {
+  "applicantOriginDetails": {
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'originData'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 10]
```diff
@@ -90,14 +91,14 @@
       "line5": "ItsTheFinalLine"
     }
   },
-  "priorResidency": [
+  "applicantPriorResidency": [
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'priorResidency'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 11]
```diff
@@ -90,14 +91,14 @@
...
     {
-      "priorStartDate": "10-10-2020",
-      "priorEndDate": "10-10-2020"
+      "startDate": "10-10-2020",
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'priorStartData'.

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 12]
```diff
@@ -90,14 +91,14 @@
...
+      "endDate": "10-10-2020"
     },
     {
-      "priorStartDate": "10-11-2020",
-      "priorEndDate": "10-11-2020"
+      "startDate": "10-11-2020",
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'priorStartData'.

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 13]
```diff
@@ -90,14 +91,14 @@
       "line5": "ItsTheFinalLine"
     }
   },
-  "priorResidency": [
+  "applicantPriorResidency": [
     {
-      "priorStartDate": "10-10-2020",
-      "priorEndDate": "10-10-2020"
+      "startDate": "10-10-2020",
+      "endDate": "10-10-2020"
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'priorEndDate'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json [hunk 14]
```diff
@@ -90,14 +91,14 @@
...
     },
     {
-      "priorStartDate": "10-11-2020",
-      "priorEndDate": "10-11-2020"
+      "startDate": "10-11-2020",
+      "endDate": "10-11-2020"
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-max-post.json]: This needs changing back to 'priorEndDate'

File: resources/public/api/conf/1.0/examples/register/register-min-post.json [hunk 15]
```diff
@@ -3,15 +3,25 @@
...
   "birthDate": "10-10-2020",
-  "birthDateVerification": "VERIFIED",
   "officeNumber": "1234",
-  "country": 1,
-  "name": {
-    "surname": "ASurname",
-    "startDate": "10-10-2020"
-  },
-  "address": {
+  "applicantNames": [
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-min-post.json]: This needs changing to 'names'

File: resources/public/api/conf/1.0/examples/register/register-min-post.json [hunk 16]
```diff
@@ -3,15 +3,25 @@
...
+    {
+    "nameType":"REGISTERED",
+    "surname": "ASurname"
+    }
+  ],
+
+  "applicantHistoricNames": [
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-min-post.json]: This needs changing to 'historicNames'

File: resources/public/api/conf/1.0/examples/register/register-min-post.json [hunk 17]
```diff
@@ -3,15 +3,25 @@
...
+    {
+      "nameType":"REGISTERED",
+      "surname": "JOHN"
+    }
+],
+  "applicantAddresses": {
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/examples/register/register-min-post.json]: This needs changing to 'addresses'

File: resources/public/api/conf/1.0/examples/register/register-max-post.json (hunk 3 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-max-post.json]: done

File: resources/public/api/conf/1.0/headers.raml (hunk 1 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/headers.raml]: removed

File: resources/public/api/conf/1.0/headers.raml (hunk 2 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/headers.raml]: removed

File: resources/public/api/conf/1.0/examples/register/register-max-post.json (hunk 4 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-max-post.json]: done

File: resources/public/api/conf/1.0/examples/register/register-max-post.json (hunk 5 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-max-post.json]: done

File: resources/public/api/conf/1.0/examples/register/register-max-post.json (hunk 6 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-max-post.json]: done

File: resources/public/api/conf/1.0/examples/register/register-max-post.json (hunk 7 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-max-post.json]: done

File: resources/public/api/conf/1.0/examples/register/register-max-post.json (hunk 13 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-max-post.json]: done

File: resources/public/api/conf/1.0/examples/register/register-min-post.json (hunk 15 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-min-post.json]: done

File: resources/public/api/conf/1.0/examples/register/register-min-post.json (hunk 17 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-min-post.json]: done

File: resources/public/api/conf/1.0/examples/register/register-min-post.json (hunk 16 above)
Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/examples/register/register-min-post.json]: done

File: resources/public/api/conf/1.0/headers.raml [hunk 18]
```diff
@@ -18,3 +18,17 @@ traits:
...
         required: true
         example: 'application/vnd.hmrc.1.0+json'
+  acceptJson:
+    headers:
+      CorrelationId:
+        description: A UUID identifier of the transaction.
+        type: string
+        required: true
+        example: 'a1e8057e-fbbc-47a8-a8b4-78d9f015c253'
+  acceptJson:
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/headers.raml]: This needs changing to originatorID as acceptJson is erroring

File: resources/public/api/conf/1.0/headers.raml [hunk 19]
```diff
@@ -18,3 +18,17 @@ traits:
         type: string
         required: true
         example: 'application/vnd.hmrc.1.0+json'
+  acceptJson:
```
Comment (TomDudley92) [inline resources/public/api/conf/1.0/headers.raml]: This needs changing to correlationID as acceptJson is erroring

Comment (DurgaJonnalagadda) [reply resources/public/api/conf/1.0/headers.raml]: done

**Pull request merged**